src/utils/connect.js

```diff
@@ -109,7 +109,7 @@ export type ConnectedProps<
 > = $ReadOnly<{|
   ...OwnProps,
   ...StateProps,
-  ...WrapDispatchProps<DispatchProps>,
+  ...DispatchProps,
 |}>;
 
 export type ConnectedComponent<
```

Comment:
Have you tried whether using directly `connect` would be OK now that they have a much improved libdef ?

Reply:
I failed at my previous attempt, this migration path worked so I think we should take it.